docs/modules/ROOT/pages/security/security.adoc

```diff
@@ -2,17 +2,13 @@
 = Overview
 
 This section describes how to deal with known or unknown security issues and
-common vulnerabilities and exploits (CVEs) related to Payara Server and
-Payara Micro.
+common vulnerabilities and exploits (CVEs) related to Payara Server Community and
+Payara Micro Community.
 
 [[download-security-fixes]]
-== Download Security Fixes
+== Security Fixes
 
-Reported security vulnerabilities by the community or Payara Services Limited's
- costumers are patched and released either in specific patch releases
- (for paid support customers exclusively) or quarterly releases. In some
- cases, we might release special hotfixes to the community to patch serious
- vulnerabilities that cannot wait for a quarterly release to be fixed.
+Reported security vulnerabilities by the Community or Payara Services Limited's costumers are patched and released in a future release. For Payara Platform Enterprise Edition, this is always the next release. Only critical security issues will be updated in the next Community Version and issues of other severity might be fixed later on.
```

Comment:
```suggestion
Security vulnerabilities reported by the user community or customers of Payara Services Ltd are patched and released in a future release. For Payara Platform Enterprise Edition, this is always the next release. Only critical security issues will likely be updated in the next Community Version and less severe issues might be fixed later on according to the time availability.
```

Reply:
The user community should be with a lowercase "c" so that it's not confused with Payara Community edition.